gp-nested-forms/gpnf-triggered-population.php

```diff
@@ -103,13 +105,38 @@ public function output_script() {
 						 * triggerFieldValue is an array
 						 */
 						if ( window.gpnf_triggered_population_entry ) {
+							// re-enable the field if entry creation gets skipped
+							$( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId ) ).removeAttr( 'disabled' );
 							return;
 						}
 
+						// serialize including disabled form data
+						$.fn.serializeWithDisabled = function() {
+							var data = {};
+
+							// iterate over all input fields, including disabled ones
+							$(this).find(':input').each(function() {
+								var name = $(this).attr('name');
+								var value = $(this).val();
+
+								if ($(this).is(':radio,:checkbox')) {
+									// for radio buttons and checkboxes, only include the selected option
+									if ($(this).is(':checked')) {
+										data[name] = value;
+									}
+								} else {
+									// for other input fields, include the value as-is
+									data[name] = value;
+								}
+							});
+
+							return $.param(data);
+						};
```

Comment:
I'm not 100% opposed to having this here, but what if you move the `$form.serialize()` call before the input gets disabled?